client/src/components/protected/Match.js

```diff
@@ -239,81 +268,121 @@ class Match extends Component {
         const info = res.info;
-        this.setState({ RS: info.RS, RF: info.RF, BS: info.BS, BF: info.BF, Host: info.Host });
+        this.setState({
+          RS: info.RS,
+          RF: info.RF,
+          BS: info.BS,
+          BF: info.BF,
+          Host: info.Host,
+        });
       } else if (currPos === this.state.userId) {
         let res = await fetch(`/matches/${this.state.matchId}/leavematch`, {
           method: "POST",
-          headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': "*" },
-          body: reqBody
-        })
+          headers: {
+            "Content-Type": "application/json",
+            "Access-Control-Allow-Origin": "*",
+          },
+          body: reqBody,
+        });
         res = await res.json();
         console.log("API setUser response", res);
         const info = res.info;
-        this.setState({ RS: info.RS, RF: info.RF, BS: info.BS, BF: info.BF, Host: info.Host });
+        this.setState({
+          RS: info.RS,
+          RF: info.RF,
+          BS: info.BS,
+          BF: info.BF,
+          Host: info.Host,
+        });
       }
-
     } catch (error) {
-      console.log('error @joingame API');
+      console.log("error @joingame API");
     }
-  }
+  };
 
   render() {
-    console.log('local state', this.state)
+    console.log("local state", this.state);
+    const { classes, setIsMatchInProgres, blueScore, redScore } = this.props;
     const {
-      classes,
-      setIsMatchInProgres,
-      blueScore,
-      redScore
-    } = this.props;
-    const { words, positionState, matchId, userId, guessesLeft, message, isOver, winner } = this.state;
+      words,
+      positionState,
+      matchId,
+      userId,
+      guessesLeft,
+      message,
+      isOver,
+      winner,
+    } = this.state;
     document.body.style.overflow = "noscroll";
-    return (<div className={classes.matchStyle}>
-      <ChatBox
-        submitHint={this.submitHint}
-        matchID={matchId}
-        userID={userId}
-        position={matchDictionary[positionState]} />
-
-      <Paper className={`${classes.paper} ${classes.centerText}`}>
-        <Typography variant="h4">{positionState}</Typography>
-        <ServerPing ping={this.ping} />
-        {["RF", "BF"].includes(matchDictionary[positionState]) ? <p>{guessesLeft} guesses left</p> : null}
-        {message !== "" ? <p>{message}</p> : null}
-        <Grid container item xs={12} className={classes.standardFlex}>
-          <MappedWords classes={classes} words={words} clickWord={this.clickWord} />
-        </Grid>
-        <Button variant="contained" color="primary" onClick={this.endFieldTurn}>End Turn</Button>
-      </Paper>
-      {isOver ? (
-        <GameOutcome
-          isOver={isOver}
-          setIsMatchInProgres={setIsMatchInProgres}
-          winner={winner}
-          blueScore={blueScore}
-          redScore={redScore}
+    return (
+      <div className={classes.matchStyle}>
```

Comment:
Per the mockups, the background here is grey and has no borders, so we shouldn't use a paper